newsrevenuehub/rev-engine
# #10661 ApplePayDomain.create management command

#### What's this PR do?
You can now run `./manage.py appledomainverify --slug my-rp-slug` to verify a RP's subdomain with Apple via Stripe.

#### How should this be manually tested?
This can only be tested in STRIPE_LIVE_MODE, since the ApplePayDomain.create command requires a live key. In Prod, you could create a rev program, then unset the "domain_apple_verified_date" field in admin. This won't "undo" the verification, but if you then run the management command it should reset the value to now.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No.
